<?xml version="1.0"?>

<launch>

    <arg name="veh" default="duckiefloat" />
    <arg name="cmd" default="cmd_vel" />
    <arg name="joy" default="joy" />

    <group ns="$(arg veh)">


        <node pkg="subt_rl" type="goal_nav_tf.py" name="goal_nav_rl" output="screen">

            <remap from="goal_in" to="/move_base_simple/goal" />
            <remap from="laser_in" to="/scan" />
            <remap from="odom_in" to="/slam_pose" />
            <remap from="joy" to="$(arg joy)" />
            <remap from="cmd_out" to="$(arg cmd)" />
        </node>

    </group>


</launch>
